src/cpp/session/modules/SessionHelp.cpp

```diff
@@ -272,13 +272,20 @@ bool handleRShowDocFile(const core::FilePath& filePath)
 }
 
 // javascript callbacks to inject into page
-const char * const kJsCallbacks =
-      "<script type=\"text/javascript\">\n"
-      "if (window.parent.helpNavigated)\n"
-      "   window.parent.helpNavigated(document, window);\n"
-      "if (window.parent.helpKeydown)\n"
-      "   window.onkeydown = function(e) {window.parent.helpKeydown(e);}\n"
-      "</script>\n";
+const char * const kJsCallbacks = R"EOF(
+<script type="text/javascript">
```

Comment:
Nit: "text/javascript" is the default in HTML5 and it is recommended to omit this, so just plain `<script>`.